### PR TITLE
Add email validity

### PR DESCRIPTION
**Pull request by Gunjuzone**
In this pull request, Javascript was used to set the validity of email input to lowercase. If a user inputs Uppercase or mixcase, a message is displayed telling the user to input the email in lower case.

### CODE REVIEW

**Review by Temmarie — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -104,3 +104,16 @@ function openModal(e) {
...
+const getInTouch = document.querySelector('.button-submit');
+
+getInTouch.addEventListener('click', () => {
+  const validatedMail = emailField.value;
+  if (!regularExpression.test(validatedMail)) {
+    emailField.setCustomValidity('Email should be in lowercase. Please input the email in lowercase!');
+  } else {
+    emailField.setCustomValidity('');
+  }
+});
```
Comment (Temmarie) [inline index.js]: - [ ] Good job adding the validations for the form, however, the email error message should show up close to the "submit" button. Kindly make this change.
- [ ] Also, you have a horizontal overflow on Desktop screens, kindly fix it.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @Gunjuzone     ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.

- [ ] It'll be nice if you could add validations for other parts of the form as well.
- [ ] Also, there's a dark border that appears in the form input field, it'll be nice if you removed it.

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Comment by Temmarie** [comment]
Hi, @Gunjuzone, I'm sorry about the initial approval, that was a mistake.  Have a good day!

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi @Gunjuzone 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**